modules/imgproc/src/sumpixels.simd.hpp

```diff
@@ -781,7 +781,7 @@ struct Integral_SIMD<uchar, double, double>
                     el4lh = v_cvt_f64_high(el4li) + prev;
                     el4hl = v_cvt_f64(el4hi) + el4ll;
                     el4hh = v_cvt_f64_high(el4hi) + el4lh;
-                    prev = vx_setall_f64(v_extract_n<v_float64::nlanes - 1>(el4hh));
+                    prev = vx_setall_f64(v_extract_n<v_float64::max_nlanes - 1>(el4hh));
```

Comment:
I think, `vx_setall_...(v_extract_n<idx>(vecreg))` is a shortcut for `v_broadcast_element<idx<(vecreg)`, so the change has the same issue

Reply:
Same as above.